Dockerfile

```diff
@@ -37,10 +37,6 @@ RUN pip install -r /api/requirements.txt --no-cache-dir
 USER root
 RUN apt-get install graphviz -y
```

Comment:
Could you also move this install up to the main `apt-get` command above please?

Reply:
good point. done